cmd/testgrid-config-generator/main.go

```diff
@@ -223,12 +225,23 @@ func main() {
 	if err != nil {
 		logrus.WithError(err).Fatalf("Failed to load Prow jobs %s", o.prowJobConfigDir)
 	}
+
+	// read the list of jobs from the allow list along with its release-type
```

Comment:
Blocking jobs must be in the release controller configuration, since it's inclusion in that config that gets them triggered in the first place, so we should validate that none of these are set to blocking here.

Reply:
I can add that check there. 
Also some of the jobs seem to be set to broken, is that something that needs to be validated?

Reply:
No, we shouldn't bother making any statement other than this file not containing blocking jobs